remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ProtobufMetadataManagerInterceptor.java

```diff
@@ -436,9 +479,13 @@ public Object visitReplaceCommand(InvocationContext ctx, ReplaceCommand command)
             }
 
             if (progressCallback != null) {
-               updateGlobalErrors(rCtx, progressCallback.getErrorFiles(), flagsBitSet);
+               CompletionStage<Void> schemaUpdate = updateSchemaErrors(rCtx, progressCallback);
```

Comment:
You can move the lines inside the if above, just like in `handlePutKeyValueResult`

Reply:
Okay, it is moved. I don't want to move the one in `visitPutMapCommand` though as it has other code in between that I am not exactly familiar with and don't want to mess with the logic this close to release.